<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Text_edit</class>
 <widget class="QMainWindow" name="Text_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>MS Sans Serif</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Text_edit</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>3</x>
      <y>6</y>
      <width>881</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Semibold</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="NewFile"/>
    <addaction name="Open"/>
    <addaction name="separator"/>
    <addaction name="save"/>
    <addaction name="saveAs"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Style</string>
    </property>
    <addaction name="Font"/>
    <addaction name="fontColour"/>
    <addaction name="bColour"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Compilation</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="About"/>
   </widget>
   <widget class="QMenu" name="menuEdit_2">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="Undo"/>
    <addaction name="Redo"/>
    <addaction name="separator"/>
    <addaction name="Cut"/>
    <addaction name="Paste"/>
    <addaction name="Copy"/>
   </widget>
   <widget class="QMenu" name="menuText_Codec">
    <property name="title">
     <string>Encoding</string>
    </property>
    <addaction name="codec1251"/>
    <addaction name="codecUTF8"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit_2"/>
   <addaction name="menuEdit"/>
   <addaction name="menu"/>
   <addaction name="menuText_Codec"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="Open"/>
   <addaction name="save"/>
   <addaction name="separator"/>
   <addaction name="Undo"/>
   <addaction name="Redo"/>
   <addaction name="separator"/>
   <addaction name="Copy"/>
   <addaction name="Cut"/>
   <addaction name="Paste"/>
   <addaction name="separator"/>
   <addaction name="About"/>
  </widget>
  <action name="Open">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/openFile.png</normaloff>:/icons/Icons/openFile.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="save">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/save48.png</normaloff>:/icons/Icons/save48.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="Font">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/font48.png</normaloff>:/icons/Icons/font48.png</iconset>
   </property>
   <property name="text">
    <string>Font</string>
   </property>
  </action>
  <action name="fontColour">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/fontColour48.png</normaloff>:/icons/Icons/fontColour48.png</iconset>
   </property>
   <property name="text">
    <string>Font colour</string>
   </property>
  </action>
  <action name="NewFile">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/newFile48.png</normaloff>:/icons/Icons/newFile48.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="fontBackgroundColour">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/colour48.png</normaloff>:/icons/Icons/colour48.png</iconset>
   </property>
   <property name="text">
    <string>Font background colour</string>
   </property>
  </action>
  <action name="bColour">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/colour48.png</normaloff>:/icons/Icons/colour48.png</iconset>
   </property>
   <property name="text">
    <string>Background colour</string>
   </property>
  </action>
  <action name="saveAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/saveAs48.png</normaloff>:/icons/Icons/saveAs48.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="Undo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/undo48.png</normaloff>:/icons/Icons/undo48.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="Redo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/redo48.png</normaloff>:/icons/Icons/redo48.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="About">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/about48.png</normaloff>:/icons/Icons/about48.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="Cut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/cut48.png</normaloff>:/icons/Icons/cut48.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="Paste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/paste48.png</normaloff>:/icons/Icons/paste48.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="Copy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/icons/Icons/copy48.png</normaloff>:/icons/Icons/copy48.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="codec1251">
   <property name="text">
    <string>Windows - 1251</string>
   </property>
  </action>
  <action name="codecUTF8">
   <property name="text">
    <string>UTF-8</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
